packages/service-clients/odsp-client/src/interfaces.ts

```diff
@@ -87,6 +91,11 @@ export interface OdspContainerServices {
 	 * Provides an object that facilitates obtaining information about users present in the Fluid session, as well as listeners for roster changes triggered by users joining or leaving the session.
 	 */
 	audience: IOdspAudience;
+
+	/**
+	 * Attach function that is used to attach the container to the ODSP service. This function is specific to the ODSP service and accepts some ODSP specific properties.
```

Comment:
Nit: I think this can be simplified, given the context (these are ODSP-specific types)

```suggestion
	 * Attaches the container to the ODSP service.
```

Reply:
Can we also add a `@param` comment for `odspProps`? What does it mean for the user not to provide anything here? We should document the default policy.